PostHog/Replay/PostHogReplayIntegration.swift

```diff
@@ -406,24 +406,6 @@
             image.imageAsset?.value(forKey: "_containingBundle") != nil
         }
 
-        // Photo library images have a UUID identifier as _assetName (e.g 64EF5A48-2E96-4AB2-A79B-AAB7E9116E3D)
-        // SF symbol and bundle images have the actual symbol name as _assetName (e.g chevron.backward)
-        private func isPhotoLibraryImage(_ image: UIImage) -> Bool {
-            guard config.sessionReplayConfig.maskPhotoLibraryImages else {
-                return false
-            }
-
-            guard let assetName = image.imageAsset?.value(forKey: "_assetName") as? String else {
-                return false
-            }
-
-            if assetName.isEmpty { return false }
-            if image.isSymbolImage { return false }
```

Comment:
should we check for `isSymbolImage` inside of `isAssetsImage`? feels like something to be kept.